src/errors/FixedToPatternError.ts

```diff
@@ -0,0 +1,13 @@
+import { Annotated } from './Annotated';
+
+export class FixedToPatternError extends Error implements Annotated {
+  specReferences = [
+    'http://hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.fixed_x_',
+    'http://hl7.org/fhir/R4/elementdefinition-definitions.html#ElementDefinition.pattern_x_'
+  ];
+  constructor(public fixedProperty: string) {
+    super(
+      `Cannot fix this element using a pattern; as it is already fixed in the StructureDefinition using ${fixedProperty}.`
```

Comment:
It might not be immediately clear to someone why they can't fix with pattern if something is already fixed with fixed. Maybe this could include something about how pattern is a looser constraint than fixed.